app/tasks/maintenance/move_image_urls_task.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+module Maintenance
+  class MoveImageUrlsTask < MaintenanceTasks::Task
+    def collection
+      Podcast.all
+    end
+
+    def process(podcast)
+      if podcast.extras["image_urls"].nil?
+        images = []
+
+        podcast.image_urls.each do |image|
```

Comment:
Should we add a nil check for podcast.image_urls ?

Reply:
We don't need this check since it returns empty active record array instead nil